lib/File/ChangeNotify/Watcher/Inotify.pm

```diff
@@ -201,10 +202,11 @@ sub _convert_event {
     return $self->event_class()->new(
         path => $event->fullname(),
         type => (
-              $event->IN_CREATE() ? 'create'
-            : $event->IN_MODIFY() ? 'modify'
-            : $event->IN_DELETE() ? 'delete'
-            :                       'unknown'
+              $event->IN_CREATE()   ? 'create'
```

Comment:
I think it'd be simpler to write `$event->IN_CREATE() or $event->IN_MOVED_TO()` here.

Reply:
No problem. I've just force-pushed a revised version with that change.